todo_app/data/session_items.py

```diff
@@ -1,78 +1,171 @@
 from flask import session
```

Comment:
It would be nice to update the file name for this module - it's no longer using/storing items in session; instead it's using Trello so adjusting it to be called `trello_items.py` would capture its intent better